Assets/SEE/Tools/Videochat/LivekitVideo.cs

```diff
@@ -0,0 +1,97 @@
+using UnityEngine;
+using Unity.Netcode;
+using SEE.Controls;
+using SEE.Utils;
+
```

Comment:
See our [Coding Guidelines](https://github.com/uni-bremen-agst/SEE/wiki/ReviewChecklist): All classes must be nested in namespaces. The nesting of the namespaces and the directories in which the source-code files reside must be the same.